docs/docs/keys/developers.md

```diff
@@ -38,7 +38,7 @@ A good choice for a remote signer is [Web3signer](https://docs.web3signer.consen
 
 The AVS can implement a feasible and sufficient method of loading the keys. This is asking for a path to a keystore folder. This keystore needs to follow some structure either the AVS knows how to read, or an AVS [plugin](/docs/category/plugin) knows how to transform to something the AVS knows how to read.
 
-Usually, the plugin idea is the most recommended because the AVS has the freedom to use whatever structure they need or want for the keystore. Also, no constraint is being put on users or key generation tools. This also fits the idea of the [AVS setup wizard tool](/docs/category/avs-setup-wizard) supporting such plugins. Apart from the wizard domain, users could use this plugin (as an AVS command/subcommand) to process the raw keystore if they go for a manual setup.
+Usually, the plugin idea is the most recommended because the AVS has the freedom to use whatever structure they need or want for the keystore. Also, no constraint is being put on users or key generation tools. 
```

Comment:
Not sure I understand what this is saying, it's a bit abstract.
Could we add info regarding current recommendation to use eigenlayer-cli directly to generate keystores fo ecdsa or bn254 keys. Anything else should use plugin since the cli doesn't support it (yet).

Reply:
yea. that makes sense.